index/hello-nrfcloud.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Hello nRF Cloud",
+    "description": "",
+    "apps": [
+        {
+            "name": "Hello nRF Cloud firmware",
+            "title": "firmware",
+            "description": "Out of Box firmware for the Thingy:91 X",
+            "kind": "sample",
+            "tags": ["dfu", "lte"],
+            "releases": [
+                {
+                    "date": "latest",
+                    "name": "Latest Release",
+                    "tag": "latest",
+                    "sdk": "latest"
```

Comment:
```suggestion
                    "date": "2024-08-11T14:37:00Z",
                    "name": "v2.0.1",
                    "tag": "v2.0.1",
                    "sdk": "v2.8.0-rc2"
```

Reply:
Not a big fan of this. We can't update the app index for every release.